### DISCUSSION TITLE
Welcome to GraphvizVS Discussions!

### DISCUSSION THREAD

**Discussion by aglasencnik** [aglasencnik/GraphvizVS#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VladD2** [comment]
How to see **a preview** for .dot file after installing your plugin?

**Reply by aglasencnik** [reply]
What do you mean by this?

**Reply by VladD2** [reply]
I installed your plugin. But it only provides syntax highlighting. From the description I understood that it allows me to display a diagram for a .dot file.

How do I do this?
